### PR TITLE
Nav bar Implementation

### PR DESCRIPTION
**Pull request by SawanKumarBhagat**

### CODE REVIEW

**Comment by maliknajeeb113** [comment]
@SawanKumarBhagat The nav bar remains expanded while switching from desktop to sm.

**Comment by SawanKumarBhagat** [comment]
> @SawanKumarBhagat The nav bar remains expanded while switching from desktop to sm.

@maliknajeeb113 Can you please attach some ss for the reference.

**Comment by maliknajeeb113** [comment]
Works better now. In mobile mode, the expanded nav does not show the current active tab.
Also the expanded nav would look better if it is rounded.

**Comment by maliknajeeb113** [comment]
Screenshot 2024-09-29 at 3 00 32 PM
Screenshot 2024-09-29 at 3 00 43 PM
What is happening here ;_;

**Comment by SawanKumarBhagat** [comment]
@maliknajeeb113  All the reported issues are fixed. Please review

File: src/App.jsx [hunk 1]
```diff
@@ -1,42 +1,49 @@
...
-export default App
+  {
+    path: "/",
+    element: <App />,
+    errorElement: <Error />,
+    children: [
+      {
+        path: "/",
+        element: (
+          <section id="home">
```
Comment (maliknajeeb113) [inline src/App.jsx]: Yaha pe section kyu dala?

**Review by maliknajeeb113 — COMMENTED** [review]
Rest all is fine.
Match the padding with the footer.

File: src/App.jsx (hunk 1 above)
Comment (SawanKumarBhagat) [reply src/App.jsx]: react-scroll ni chalega bina sections ke, ya to components hi div ki jgah sections me wrap kr ke usme id set kr lo

**Comment by SawanKumarBhagat** [comment]
> Rest all is fine. Match the padding with the footer.

Which padding?

**Comment by maliknajeeb113** [comment]
Here

Screenshot 2024-11-20 at 12 24 12 AM


> > Rest all is fine. Match the padding with the footer.
> 
> Which padding?

**Comment by SawanKumarBhagat** [comment]
@maliknajeeb113 Matched the padding with the footer and moved the sections to the component level.

**Review by maliknajeeb113 — COMMENTED** [review]
Cool. Approving the same.

**Pull request merged**